test/init.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+oneTimeSetUp() {
+  BASE=~/.lc_init_test
+  rm -rf "$BASE"
+  mkdir -p "$BASE"
+
+  SERVER="${BASE}/server"
+  SERVER_LOG="${BASE}/tendermint.log"
+
+  tendermint init --home="$SERVER" >> "$SERVER_LOG"
+  if ! assertTrue $?; then return 1; fi
+
+  GENESIS_FILE=${SERVER}/genesis.json
+  CHAIN_ID=$(cat ${GENESIS_FILE} | jq .chain_id | tr -d \")
+
+  printf "starting tendermint...\n"
+  tendermint node --proxy_app=dummy --home="$SERVER" >> "$SERVER_LOG" 2>&1 &
+  sleep 5
+  PID_SERVER=$!
+  disown
+  if ! ps $PID_SERVER >/dev/null; then
+    echo "**STARTUP FAILED**"
+    cat $SERVER_LOG
+    return 1
+  fi
+}
+
+oneTimeTearDown() {
+  printf "\nstoping tendermint..."
+  kill -9 $PID_SERVER >/dev/null 2>&1
+  sleep 1
+}
+
+test01goodInit() {
+  export TMHOME=${BASE}/client-01
+  assertFalse "ls ${TMHOME} 2>/dev/null >&2"
+
+  echo y | tmcli init --node=tcp://localhost:46657 --chain-id="${CHAIN_ID}" > /dev/null
+  assertTrue "initialized light-client" $?
+  checkDir $TMHOME 3
+}
+
+test02badInit() {
+  export TMHOME=${BASE}/client-02
+  assertFalse "ls ${TMHOME} 2>/dev/null >&2"
+
+  # no node where we go
+  echo y | tmcli init --node=tcp://localhost:9999 --chain-id="bad-chain-id" > /dev/null 2>&1
```

Comment:
Not that it matters here but we should only change one thing at a time. ie. we should use the correct chain-id if we're using the wrong node